apps/prairielearn/src/lib/db-types.ts

```diff
@@ -491,6 +491,24 @@ export const FileEditSchema = z.object({
 });
 export type FileEdit = z.infer<typeof FileEditSchema>;
 
+export const GenerationThreadItemSchema = z.object({
+  completion: z.any(),
+  context: z.string().nullable(),
+  errors: z.array(z.string()),
+  html: z.string().nullable(),
+  id: z.string(),
+  prompt_type: z.enum(['initial_prompt', 'human_revision', 'autorevision']),
+  prompting_user: z.string(),
+  python: z.string().nullable(),
+  question_id: z.string(),
+  response: z.string(),
+  title: z.string().nullable(),
+  user_prompt: z.string(),
+  uuid: z.string().nullable(),
+});
```

Comment:
What table is this for? `ai_generation_prompts`? Please name things consistently.